tests/unit/test_authentication.py

```diff
@@ -107,6 +107,14 @@ def test_simple_oauth_token_provider_returns_configured_token() -> None:
     assert token_provider.token() == "TOKEN"
 
 
+def test_simple_oauth_token_provider_returns_configured_token_and_expiry() -> None:
```

Comment:
question (non-blocking): Is this related to this PR?

Reply:
Yes, the method tested here is used in the new admin client (slight difference between the expected "fetch token" interface).